x-pack/plugins/apm/server/routes/alerts/rule_types/anomaly/register_anomaly_rule_type.ts

```diff
@@ -281,11 +280,12 @@ export function registerAnomalyRuleType({
 
           const severityLevel = getSeverity(score);
           const reasonMessage = formatAnomalyReason({
-            measured: score,
+            anomalyScore: score,
             serviceName,
             severityLevel,
             windowSize: params.windowSize,
             windowUnit: params.windowUnit,
+            anomalyDetectorTypes: params.anomalyDetectorTypes,
```

Comment:
As we group by `detector_index`, the anomalyDetectorType(s) values should correspond to each individual anomaly.

We could have transaction latency and failed transaction rate having anomalies with different scores. In the reason message the user should be able to identify which detector has this score.

Reply:
fixed here https://github.com/elastic/kibana/pull/171901/commits/39de1312c108d91c03dfd530f0adb4423d80f52f